Code/GraphMol/Conformer.h

```diff
@@ -115,10 +115,14 @@ class RDKIT_GRAPHMOL_EXPORT Conformer {
 
   //! Get the number of atoms
   inline unsigned int getNumAtoms() const { return rdcast<unsigned int>(d_positions.size()); }
-
+  inline bool hasZCoords() const {
+    for(auto p: d_positions){ if (p.z != 0.0) return true; }
```

Comment:
Seems like we should do this with `feq()` instead of `!=`?

Reply:
I think this should be exact equivalence and not using a tolerance.

Reply:
0.0 is exact in floating point

Reply:
how about hasNonZeroZ ?